keys_storage.go

```diff
@@ -26,20 +27,21 @@ type KeysStorage interface {
 	Count(k Key) (uint, error)
 
 	// All returns all the keys
-	All() (map[Key]map[uint]Key, error)
+	All() (map[string]map[uint]Key, error)
```

Comment:
Why do we need to change this?

Reply:
we use slices now and can't be used as indexes for maps, it s only used for testing